integration_tests/dbt_project.yml

```diff
@@ -33,11 +33,6 @@ seeds:
         ad_group_id: "{{ 'INT64' if target.type == 'bigquery' else 'bigint' }}"
         campaign_id: "{{ 'INT64' if target.type == 'bigquery' else 'bigint' }}"
         external_customer_id: "{{ 'INT64' if target.type == 'bigquery' else 'bigint' }}"
-    google_ads_final_url_performance_data:
```

Comment:
Were those duplicated on purpose?

Reply:
I do not believe they were. Thanks for catching this and removing the duplicate 😄